tmt/base.py

```diff
@@ -1707,15 +1964,18 @@ def stories(self, keys=None, names=None, filters=None, conditions=None,
             filters, conditions, links, excludes)
 
     @staticmethod
-    def init(path, template, force, **kwargs):
+    def init(path: str, template: str, force: bool, **kwargs: Any) -> None:
         """ Initialize a new tmt tree, optionally with a template """
         path = os.path.realpath(path)
         dry = Tree._opt('dry')
 
         # Check for existing tree
+        tree: Optional[Tree] = None
+
         try:
             tree = tmt.Tree(path)
```

Comment:
```suggestion
            tree = Tree(path)
```

Reply:
Fixed in 3017568a8eed9792f95bd9528d2c0a22cedd84a6.